### ISSUE TITLE
Issue with 5.6.0 when archiving - CocoaPods

### ISSUE THREAD

**Issue by Legoless** [smooch/smooch-ios#147]
Hello, I updated Smooch to **5.6.0** today using CocoaPods, but our archive scripts started failing with this error:

```
 ✗ ld: bitcode bundle could not be generated because '/tmp/sandbox/workspace/Application/Pods/Smooch/Smooch.framework/Smooch' was built without full bitcode. All frameworks and dylibs for bitcode must be generated from Xcode Archive or Install build for architecture armv7
2833
    ✗ clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

It looks like the new `Smooch.framework` is not compiled with bitcode, which causes our builds to break.

**Comment by mspensieri** [comment]
Thanks for the report, the latest release does seem to be missing bitcode (the zip file is 5.5MB smaller than the previous release). We'll investigate why this would be the case. 

In the meantime, v5.5.2 should be working correctly

**Comment by mspensieri** [comment]
Looks like the update to XCode 8.3 changed something in our build process. We've just released [v5.6.1](https://github.com/smooch/smooch-ios/releases/tag/5.6.1) that should solve this issue
